static/compiler.js

```diff
@@ -1416,10 +1424,27 @@ Compiler.prototype.updateLibsDropdown = function () {
                             })
                         )
                     );
+                    if (version.used) {
+                        anyVerInUse = true;
+                    }
+                });
+
+                libArrow.css('color', anyVerInUse ? 'green' : 'black');
+
+                libGroup.hide();
```

Comment:
subdropdown states are not preserved between dropdown toggles/state saves (I dont think it's needed, that's why it's not here. If someone thinks otherwise, please let me know)